frontend/pages/hosts/ManageHostsPage/components/HostsFilterBlock/HostsFilterBlock.tsx

```diff
@@ -531,13 +531,6 @@ const HostsFilterBlock = ({
         <FilterPill
           label={`OS settings: ${configProfile?.name}`}
           onClear={() => handleClearFilter(["profile_status", "profile_uuid"])}
-          tooltipDescription={
-            <>
-              OS settings:
-              <br />
-              {configProfile?.name}
-            </>
-          }
```

Comment:
why the removal here?

Reply:
Notice it's the same as the passed in label - the intention was to have `TooltipTruncatedText` behavior here, which the new fallback behavior now takes care of

Reply:
Ah, gotcha.  Thanks!